README.rst

```diff
@@ -163,13 +163,17 @@ Add routes for websockets:
 
 .. code-block:: python
 
-    # In yourproject/routing.py
+    # In yourproject/asgi.py
     from channels.auth import AuthMiddlewareStack
     from channels.routing import ProtocolTypeRouter, URLRouter
+    from django.core.asgi import get_asgi_application
     from openwisp_notifications.websockets.routing import get_routes
 
     application = ProtocolTypeRouter(
-        {'websocket': AuthMiddlewareStack(URLRouter(get_routes()))}
+        {
+            "http": get_asgi_application(),
```

Comment:
I don't think `http` will be needed. We are not handling HTTP requests with ASGI application.

Reply:
> When this is merged, we would need to update django loci as well before releasing the next version of openwisp-notifications, otherwise the ansible-openwisp2 installs will fail! 😄
> 
> @niteshsinha17 are you interested to take that one as well? 😄

I haven't installed it yet. But I will install and try to upgrade it too.

Reply:
Okay, I will remove it.